src/main/java/com/epam/ta/reportportal/core/widget/content/CasesTrendContentLoader.java

```diff
@@ -157,4 +160,22 @@
 		result.put(RESULT, initial);
 		return result;
 	}
+
+	/**
+	 * Revert results if in descending order as it is a trend chart.
+	 *
+	 * @param input - callback output result
+	 * @return - transformed Map with reverse ordered elements
+	 */
+	private Map<String, List<ChartObject>> mapRevert(Map<String, List<ChartObject>> input, Sort sorting) {
+		Sort.Order order = sorting.iterator().next();
+		if (order.getProperty().equalsIgnoreCase(SORT_FIELD) && order.isAscending()) {
+			return input;
+		}
+		return input.entrySet().stream().collect(MoreCollectors.toLinkedMap(Map.Entry::getKey, it -> {
```

Comment:
does this map immutable or we can just reverse all map items without creating new map?